### PR TITLE
update: update CocoaPods integration for 1.4.20 release

### PR DESCRIPTION
**Pull request by koshachy**

### CODE REVIEW

File: COCOAPODS.md [hunk 1]
```diff
@@ -187,15 +228,365 @@ import cocoapods.subspec_dependency.*
...
+### Add a dependency on a Pod library with custom C interoperability options
+
+You can add dependencies on a Pod library with custom C interoperability options with `pod()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of a Pod library in the `pod()` function.
+In the configuration block specify the C interoperability options:
+
+    * `extraOpts` – to specify the list of options to a Pod library. For example, specific flags
+    * `packageName` – to specify the package name. If you specify it, you can import the library using the package name: `import cocoapods.<packageName>`
```
Comment (ilmat192) [inline COCOAPODS.md:437]: AFAIK the plugin doesn't add the `cocoapods` prefix and just takes the value provided by a user.

File: COCOAPODS.md [hunk 2]
```diff
@@ -187,15 +228,365 @@ import cocoapods.subspec_dependency.*
...
+### Add a dependency on a Pod library from an archive
+
+You can add dependencies on a Pod library from `zip`, `tar`, or `jar` archive with `pod()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of a Pod library in the `pod()` function.
+In the configuration block specify the path to the archive: use the `url()` function with an arbitrary HTTP address in the `source` parameter value. 
+
+    Additionally, you can specify the Boolean `flatten` parameter as a second argument for the `url()` function.
+    It indicates that all Pod files located in the root directory.
```
[the remark this answers is not in the document]
Comment (koshachy) [reply COCOAPODS.md]: fixed

File: COCOAPODS.md [hunk 3]
```diff
@@ -96,39 +94,62 @@ the build process of an Xcode project.
...
-            pod("SDWebImage/MapKit")
-        }
-    }
-    ```
-    
-    </div>
+   > You can add dependencies on subspecs.
+   {:.note}
+
+2. Specify the minimum target version for the Pod library.
```
[the remark this answers is not in the document]
Comment (koshachy) [reply COCOAPODS.md]: fixed

File: COCOAPODS.md [hunk 4]
```diff
@@ -1,18 +1,16 @@
...
+
 Use Xcode only when you need to write Swift/Objective-C code or run your application on a simulator or device.
+To work correctly with Xcode, you should [update you Podfile](#update-podfile-for-xcode). 
  
-Depending on your project and purposes, you can add dependencies between:
-* [A Kotlin project and a Pod library from the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository)
-* [A Kotlin project and a Pod library stored locally](#add-a-dependency-on-a-pod-library-stored-locally)
-* [A Kotlin Pod and an Xcode project with one target](#add-a-dependency-between-a-kotlin-pod-and-xcode-project-with-one-target) 
-or [several targets](#add-a-dependency-between-a-kotlin-pod-with-an-xcode-project-with-several-targets)
+Depending on your project and purposes, you can add dependencies between [a Kotlin project and a Pod library](#add-dependencies-on-pod-libraries), or [a Kotlin Pod and an Xcode project](#use-a-kotlin-gradle-project-as-a-cocoapods-dependency).
```
[the remark this answers is not in the document]
Comment (koshachy) [reply COCOAPODS.md]: I rewrote it to "as well as". Is it ok?

File: COCOAPODS.md [hunk 5]
```diff
@@ -187,15 +228,365 @@ import cocoapods.subspec_dependency.*
...
+### Add a dependency on a Pod library from the Git repository
+
+You can add dependencies on a Pod library from a custom Git repository with `pod()` to `build.gradle.kts` 
+(`build.gradle`) of your project: 
+
+1. Specify the name of a Pod library in the `pod()` function.
+In the configuration block specify the path to the git repository: use the `git()` function in the `source` parameter value.
+
+    Additionally, you can specify the following parameters in the block after `git()`:
+    * `branch` – to use a different branch of the repository
```
[the remark this answers is not in the document]
Comment (zoldater) [reply COCOAPODS.md:241]: @ilmat192 if commit, tag or branch weren't specified current master's HEAD is to be used. The order is defined as follows: commit -> tag -> branch -> HEAD (as default)

File: COCOAPODS.md (hunk 1 above)
Comment (zoldater) [reply COCOAPODS.md:437]: @koshachy Right, in the case of using custom `packageName` value import statement won't contain `cocoapods.` prefix

Comment (zoldater) [reply COCOAPODS.md:437]: `cocoapods.<libraryName>` is default `packageName`

File: COCOAPODS.md [hunk 6]
```diff
@@ -187,15 +228,365 @@ import cocoapods.subspec_dependency.*
...
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+If you use the `packageName` parameter, you can import the library using the package name: `import cocoapods.<packageName>`:
```
Comment (zoldater) [inline COCOAPODS.md:480]: Needs to be changed to `import <packageName>`

File: COCOAPODS.md [hunk 7]
```diff
@@ -187,15 +228,365 @@ import cocoapods.subspec_dependency.*
...
+### Add a dependency on a static Pod library
+
+You can add dependencies on a static Pod library with `pod()` and `useLibraries()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of the library using the `pod()` function.
+
+2. Call the `useLibraries()` function: it enables a special flag for static libraries. 
+
+   > You can add a dependency on a static library from a repository or stored locally.
```
Comment (zoldater) [inline COCOAPODS.md:499]: There are no restrictions, git, archive or custom specRepo pods are also supported

File: COCOAPODS.md [hunk 8]
```diff
@@ -187,15 +228,365 @@ import cocoapods.subspec_dependency.*
...
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+### Update Podfile for Xcode
+
+Kotlin requires Podfile changes to work correctly with Xcode:
```
Comment (zoldater) [inline COCOAPODS.md:546]: This step requires only if user has his own Xcode project. If Xcode project exists and `podfile` is changed to actual path, absence of this information will fail `podInstall` task with diagnostic message about possible lost paths in`Podfile`

File: COCOAPODS.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ilmat192) [reply COCOAPODS.md]: I think yes

File: COCOAPODS.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ilmat192) [reply COCOAPODS.md:241]: > if commit, tag or branch weren't specified current master's HEAD is to be used. The order is defined as follows: commit -> tag -> branch -> HEAD (as default)

I think that It worth mentioning this in the doc.

[the remark this answers is not in the document]
Comment (koshachy) [reply COCOAPODS.md]: Added info on the `HEAD` and the order.

File: COCOAPODS.md (hunk 1 above)
Comment (koshachy) [reply COCOAPODS.md]: Fixed in two places.

File: COCOAPODS.md (hunk 6 above)
Comment (koshachy) [reply COCOAPODS.md:483]: Fixed.

File: COCOAPODS.md (hunk 7 above)
Comment (koshachy) [reply COCOAPODS.md]: Deleted the note.

File: COCOAPODS.md (hunk 8 above)
Comment (koshachy) [reply COCOAPODS.md]: Updated the intro and add an additional sentence on error messages.

File: COCOAPODS.md (hunk 6 above)
Comment (zoldater) [reply COCOAPODS.md:483]: Not at all:
If you use the `packageName` parameter, you can import the library using the package name: `import cocoapods.<packageName>`:

Comment (koshachy) [reply COCOAPODS.md]: Oh, my bad. Fixed!

**Review by zoldater — APPROVED** [review]

**Review by wild-lynx — APPROVED** [review]

**Review by ilmat192 — APPROVED** [review]

File: COCOAPODS.md [hunk 9]
```diff
@@ -96,39 +94,62 @@ the build process of an Xcode project.
...
-You can add dependencies between a Kotlin project and Pod libraries [stored in the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
-and [stored locally](#add-a-dependency-on-a-pod-library-stored-locally).
+To add dependencies between a Kotlin project and a Pod library, you should [complete the initial configuration](#install-the-cocoapods-dependency-manager-and-plugin).
+This allows you to add dependencies on the following types of Pod libraries: 
+ * [A Pod library from a CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
+ * [A Pod library stored locally](#add-a-dependency-on-a-pod-library-stored-locally)
+ * [A Pod library from a Git repository](#add-a-dependency-on-a-pod-library-from-the-git-repository)
+ * [A Pod library from an archive](#add-a-dependency-on-a-pod-library-from-an-archive)
+ * [A Pod library from a custom Spec repository](#add-a-dependency-on-a-pod-library-from-a-custom-spec-repository)
+ * [A Pod library with custom C Interop options](#add-a-dependency-on-a-pod-library-with-custom-c-interoperability-options)
```
Comment (katia-energizer) [inline COCOAPODS.md:104]: I think we should use consistent terminology. In current docs, I see cinterop as one word, not capitalized

File: COCOAPODS.md [hunk 10]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+    * `commit` – to use a specific commit of the repository
+    * `tag` – to use a specific tag of the repository
+    * `branch` – to use a specific branch of the repository
+
+    The `git()` function prioritizes passed parameters in the following order (from highest to lowest): `commit`, `tag`, `branch`.
+    If you don't specify any parameter, Kotlin plugin uses `HEAD` of the `master` branch.
+
+2. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
```
Comment (katia-energizer) [inline COCOAPODS.md:250]: why "may"? what does it depend on?

File: COCOAPODS.md [hunk 11]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+
+### Add a dependency on a Pod library from an archive
+
+You can add dependencies on a Pod library from `zip`, `tar`, or `jar` archive with `pod()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of a Pod library in the `pod()` function.
+In the configuration block specify the path to the archive: use the `url()` function with an arbitrary HTTP address in the `source` parameter value. 
+
+    Additionally, you can specify the Boolean `flatten` parameter as a second argument for the `url()` function.
```
Comment (katia-energizer) [inline COCOAPODS.md:314]: Do you need to capitalize Boolean here?

File: COCOAPODS.md [hunk 12]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+
+### Add a dependency on a Pod library with custom C interoperability options
```
Comment (katia-energizer) [inline COCOAPODS.md:431]: Again. about cinterop | C Interop | C interoperability options. In other docs, blog posts, and what's new, we used cinterop.

File: COCOAPODS.md [hunk 13]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+
+### Add a dependency on a Pod library from a custom Spec repository
```
Comment (katia-energizer) [inline COCOAPODS.md:375]: What's a difference between Spec and Podspec? Are they the same?

File: COCOAPODS.md [hunk 14]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+   ```ruby
+   target 'ios-app' do
+       # ... other depedencies ...
+       pod 'podspecWithFilesExample', :path => '../kotlin-libr/build/cocoapods/externalSources/url/podspecWithFilesExample'
+   end
+   ```
+
+   </div>
+
+* When you add a library from the custom Spec repository, you should also specify the [location](https://guides.cocoapods.org/syntax/podfile.html#source) of specs at the beginning of your Podfile:
```
Comment (katia-energizer) [inline COCOAPODS.md:563]: You capitalize Spec in some cases, but not in all. Is there a reason?

File: COCOAPODS.md [hunk 15]
```diff
@@ -212,8 +605,8 @@ directive.
 2. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` to `build.gradle.kts` (`build.gradle`)
```
Comment (katia-energizer) [inline COCOAPODS.md:605]: Could you add this as an open question to discuss on our sync – how we format `build.gradle.kts` (`build.gradle`)?
In JS docs, we format as `build.gradle`(`.kts`). Let's agree on consistent formatting.

**Review by katia-energizer — APPROVED** [review]
Overall, the topic has become even longer and requires restructuring.
There are some comments on formatting and naming. Once they are resolved, I am OK. Restructuring can be done later if there is no time now.

File: COCOAPODS.md [hunk 16]
```diff
@@ -1,18 +1,16 @@
...
+and completion. You can build the whole Kotlin project with Gradle and not ever have to switch to Xcode. 
+
 Use Xcode only when you need to write Swift/Objective-C code or run your application on a simulator or device.
- 
-Depending on your project and purposes, you can add dependencies between:
-* [A Kotlin project and a Pod library from the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository)
-* [A Kotlin project and a Pod library stored locally](#add-a-dependency-on-a-pod-library-stored-locally)
-* [A Kotlin Pod and an Xcode project with one target](#add-a-dependency-between-a-kotlin-pod-and-xcode-project-with-one-target) 
-or [several targets](#add-a-dependency-between-a-kotlin-pod-with-an-xcode-project-with-several-targets)
+To work correctly with Xcode, you should [update you Podfile](#update-podfile-for-xcode).
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:11]:
```suggestion
To work correctly with Xcode, you should [update your Podfile](#update-podfile-for-xcode). 
```

File: COCOAPODS.md [hunk 17]
```diff
@@ -96,39 +94,62 @@ the build process of an Xcode project.
 
 ## Add dependencies on Pod libraries
 
-You can add dependencies between a Kotlin project and Pod libraries [stored in the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
-and [stored locally](#add-a-dependency-on-a-pod-library-stored-locally).
+To add dependencies between a Kotlin project and a Pod library, you should [complete the initial configuration](#install-the-cocoapods-dependency-manager-and-plugin).
+This allows you to add dependencies on the following types of Pod libraries: 
+ * [A Pod library from a CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository)
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:99]:
```suggestion
 * [A Pod library from the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
```

File: COCOAPODS.md [hunk 18]
```diff
@@ -96,39 +94,62 @@ the build process of an Xcode project.
...
 
-2. Re-import the project.
+           ios.deploymentTarget = "13.5"
+
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           pod("AFNetworking") {
+                version = "~> 4.0.1"
+           }
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:145]: No `SDWebImage` anymore, so it should probably be removed from the imports below.

File: COCOAPODS.md [hunk 19]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+
+3. Re-import the project.
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:299]:
```suggestion
import cocoapods.JSONModel.*
```

File: COCOAPODS.md [hunk 20]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+### Add a dependency on a Pod library from an archive
+
+You can add dependencies on a Pod library from `zip`, `tar`, or `jar` archive with `pod()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of a Pod library in the `pod()` function.
+In the configuration block specify the path to the archive: use the `url()` function with an arbitrary HTTP address in the `source` parameter value. 
+
+    Additionally, you can specify the Boolean `flatten` parameter as a second argument for the `url()` function.
+    It indicates that all Pod files located in the root directory of the archive.
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:315]:
```suggestion
    It indicates that all Pod files are located in the root directory of the archive.
```

File: COCOAPODS.md [hunk 21]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+>
+{:.note} 
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:368]:
```suggestion
import cocoapods.pod_dependency.*
```

**Comment by SvyatoslavScherbina** [comment]
@wild-lynx @katia-energizer There is a couple of abandoned PRs to the same document.
https://github.com/JetBrains/kotlin-native/pull/4399
https://github.com/JetBrains/kotlin-native/pull/4390

File: COCOAPODS.md [hunk 22]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+```
+
+</div>
+
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:373]: The link might be more specific.
Same for other sample links in the document.

File: COCOAPODS.md [hunk 23]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+> For example, `source 'https://github.com/Kotlin/kotlin-cocoapods-spec.git'`
+{:.note} 
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:424]:
```suggestion
import cocoapods.example.*
```

File: COCOAPODS.md [hunk 24]
```diff
@@ -96,39 +94,62 @@ the build process of an Xcode project.
...
-and [stored locally](#add-a-dependency-on-a-pod-library-stored-locally).
+To add dependencies between a Kotlin project and a Pod library, you should [complete the initial configuration](#install-the-cocoapods-dependency-manager-and-plugin).
+This allows you to add dependencies on the following types of Pod libraries: 
+ * [A Pod library from a CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
+ * [A Pod library stored locally](#add-a-dependency-on-a-pod-library-stored-locally)
+ * [A Pod library from a Git repository](#add-a-dependency-on-a-pod-library-from-the-git-repository)
+ * [A Pod library from an archive](#add-a-dependency-on-a-pod-library-from-an-archive)
+ * [A Pod library from a custom Spec repository](#add-a-dependency-on-a-pod-library-from-a-custom-spec-repository)
+ * [A Pod library with custom C Interop options](#add-a-dependency-on-a-pod-library-with-custom-c-interoperability-options)
+ * [A static Pod library](#add-a-dependency-on-a-static-pod-library)
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:105]: These last two sections seem to be orthogonal to the previous ones. However, the way this list is organized here and the contents of these sections don't help to understand that.

File: COCOAPODS.md [hunk 25]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+
+3. Re-import the project.
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:478]: Wrong imports.

File: COCOAPODS.md [hunk 26]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+   
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           pod("new_pod") {
+               extraOpts = listOf("-compiler-option")
+               packageName = "package"
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:461]: Please make sure this actually works, and you are able to import from a package named "`package`" in Kotlin.

File: COCOAPODS.md [hunk 27]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+
+4. Re-import the project.
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:539]: Wrong imports

File: COCOAPODS.md [hunk 28]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+```
+
+</div>
+
+### Update Podfile for Xcode
+
+If you already have the Xcode project, CocoaPods plugin requires Podfile changes to work correctly:
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:546]: > If you already have the Xcode project

Seems awkward. What if I don't have the Xcode project, but going to create one?
What if I do, but not going to use it?

This should probably be more like "If you would like to import your Kotlin project to an Xcode project,".

File: COCOAPODS.md [hunk 29]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+
+* If your project has any Git, HTTP, custom Spec repository dependency, you should also specify the path to the Podspec in the Podfile.
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:548]: Does this list of cases match the list of sections that actually mention this required change to the Podfile?

File: COCOAPODS.md [hunk 30]
```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
...
+
+   For example, if you add the dependency on `podspecWithFilesExample`, declare the path to the Podspec in the Podfile:
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+   
+   ```ruby
+   target 'ios-app' do
+       # ... other depedencies ...
+       pod 'podspecWithFilesExample', :path => '../kotlin-libr/build/cocoapods/externalSources/url/podspecWithFilesExample'
```
Comment (SvyatoslavScherbina) [inline COCOAPODS.md:557]: What is this path?

File: COCOAPODS.md (hunk 13 above)
Comment (koshachy) [reply COCOAPODS.md:374]: Yep, according to the CocoaPods docs they are the same. I double-checked all usages of this term, thanks.

File: COCOAPODS.md (hunk 18 above)
Comment (koshachy) [reply COCOAPODS.md:145]: I removed them.

File: COCOAPODS.md (hunk 22 above)
Comment (koshachy) [reply COCOAPODS.md:389]: Do you mean links to the specific code lines? I'm not sure, because we talking about the whole project, not the 'usage example'.

**Review by koshachy — COMMENTED** [review]
@SvyatoslavScherbina  I rewrote some parts taking into account your comments.
I planned to revise the structure of the document after the release: https://youtrack.jetbrains.com/issue/KT-43363

File: COCOAPODS.md (hunk 24 above)
Comment (koshachy) [reply COCOAPODS.md]: I planned to restructure after the upcoming release: https://youtrack.jetbrains.com/issue/KT-43363

File: COCOAPODS.md (hunk 26 above)
Comment (koshachy) [reply COCOAPODS.md]: I replace the example with another one.

File: COCOAPODS.md (hunk 14 above)
Comment (koshachy) [reply COCOAPODS.md]: It was according to CocoaPods docs: they use spec when it's about sources in Podfile (and Spec or Podspec when it's about repo).

File: COCOAPODS.md (hunk 29 above)
Comment (koshachy) [reply COCOAPODS.md]: I checked and added notes in the corresponding sections.
